themes/default/index.html

```diff
@@ -248,8 +248,6 @@
                   <td class="content-cell">
                     <% if (locals.title) { %>
```

Comment:
I see what you mean there, but currently, we pretty much assume all mails generated have either a `title` or `greeting` + `name`, not sure if we'd ever be in a situation where no title/greeting is intended. In any case, should be pretty easy to add support for in a different PR. =)